src/main/java/org/terasology/climateConditions/BodyTemperatureValueChangedEvent.java

```diff
@@ -0,0 +1,25 @@
+// Copyright 2020 The Terasology Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+package org.terasology.climateConditions;
+
+import org.terasology.entitySystem.event.Event;
+
+public class BodyTemperatureValueChangedEvent implements Event {
+    private float oldValue;
+    private float newValue;
+
+
+    public BodyTemperatureValueChangedEvent(float oldValue, float newValue) {
+        this.oldValue = oldValue;
+        this.newValue = newValue;
+    }
+
+    public float getNewBodyTemperatureValue() {
```

Comment:
As this event is called `BodyTemperatureValueChangedEvent` getter and setter could be named simpler imho:

```suggestion
    public float getNewValue() {
```